_data/code/municipal/city-of-sault-ste-marie-on.yml

```diff
@@ -488,8 +488,8 @@ releases:
           fr: 'https://github.com/cityssm/lot-occupancy-system/blob/main/LICENSE'
         spdxID: MIT
         level:
-          en: Main license
-          fr: License principale
+          en: ''
+          fr: !<tag:yaml.org,2002:js/undefined> ''
```

Comment:
No license changes. Guess the current details didn't load into the form.

Reply:
I fixed the issue.  I need to look into the cause of that bug.  Thanks for sending your changes!